components/permit-holders/modals/EditUserInformationModal.tsx

```diff
@@ -47,17 +57,53 @@ export default function EditUserInformationModal({ children }: EditUserInformati
 
   //   TODO: Add error states for each field (post-mvp)
 
-  const successfulEditToast = useToast();
+  const toast = useToast();
 
-  const handleSubmit = (event: SyntheticEvent) => {
-    event.preventDefault();
-    // TODO: Will be addressed in API hookup
-    onClose();
+  // Submit edited user information mutation
+  const [submitEditedUserInformation, { loading }] = useMutation<
+    UpdateApplicantResponse,
+    UpdateApplicantRequest
+  >(UPDATE_APPLICANT_MUTATION, {
+    onCompleted: data => {
+      if (data?.updateApplicant.ok) {
+        onClose();
+        toast({
+          render: () => (
+            <SuccessfulEditAlert>{"User's information has been edited."}</SuccessfulEditAlert>
+          ),
```

Comment:
What's the reason for not just using the normal `toast` parameters (status, description) as is done below for the error case?

Reply:
@j-bunk

Reply:
Hiyo! I don't have my laptop (writing this on my phone) so I'm having a hard time navigating the code and retracing to figure out why i did that. If I remember correctly it was because I wanted to match the figma designs which looked different to the default toast. I can't recall why I didn't customize the Chakra theming (maybe the default is required somewhere else as is?) We also have another custom built toast `IncompleteSectionAlert`. Sorry can't really dig into this without my computer hope this helps!